.github/workflows/terraformApply.yml

```diff
@@ -83,3 +87,91 @@ jobs:
         run: "TF_LOG=info terraform apply -auto-approve -input=false ${{ env.terraform_directory }}"
         env:
           VAULT_TOKEN: ${{ steps.token.outputs.vault_token }}
+  notify_deploy_start:
+    name: Announce deployments starting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch Slack token
+        id: vault-lookup
+        uses: hashicorp/vault-action@v2.1.2
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.MONSTER_AUTH_ROLE_ID }}
+          secretId: ${{ secrets.MONSTER_AUTH_ROLE_SECRET }}
+          secrets: secret/dsde/monster/dev/slack-notifier oauth-token | slack_token
```

Comment:
Ugh, this is annoying. We should true these up but there are other slack hooks depending on these secrets.

Reply:
not sure i follow what you mean by 'true these up'.

Reply:
Standardize on a single key name for slack tokens.